reference/ToDo/src/ToDo/Views/Dialogs/SettingsFlyout.xaml.cs

```diff
@@ -10,13 +10,13 @@ public sealed partial class SettingsFlyout : Flyout, IRecipient<ThemeChangedMess
     private bool _isThemeInitialized = false;
 #endif
 
-	public SettingsFlyout()
-	{
-		this.InitializeComponent();
+    public SettingsFlyout()
+    {
+        this.InitializeComponent();
         WeakReferenceMessenger.Default.Register(this);
     }
 
-    private void ThemeChipGroup_ItemChecked(object sender, ChipItemEventArgs e)
+    private void ThemeChipGroup_ItemClick(object sender, ChipItemEventArgs e)
```

Comment:
I had changed it to `ItemChecked` because if you click on a theme that's already selected it will change the theme anyway. I think this was on Windows only if I remember correctly?

Example: 
1) Select light theme (theme changes to light)
2) Click on light theme again (theme changes to dark)

Just verify that this is not the case anymore

Reply:
@rajamatt thank you for the feedback. I will need to check back and sync with you as there is another issue with `ItemCheck` (windows only also normally) where if you open/close the settings flyout multiple times without clicking on any of the chips to switch the theme, for some reason the theme is changed anyway every time we open again the flyout. I will be able to show you quickly the behavior next week so we can find a proper solution and open appropriate related issues if any.

Reply:
@rajamatt I undone my changes and I will open a proper issue for this one so we can better investigate, as after doing some additional tests I can see issues with ItemChecked or ItemClick